src/Dfe.PlanTech.Web/Controllers/CheckAnswersController.cs

```diff
@@ -16,30 +16,78 @@ public class CheckAnswersController : BaseController<CheckAnswersController>
 {
     public const string PAGE_SLUG = "check-answers";
 
-    public CheckAnswersController(ILogger<CheckAnswersController> logger) : base(logger)
+    public const string INLINE_RECOMMENDATION_UNAVAILABLE_ERROR_MESSAGE =
```

Comment:
Can you make this lower case please.  New way is to keep constants camelcase.  We are using a mixture so lets stick to one.